docs/pages/admin-guides/management/admin/uninstall-teleport.mdx

```diff
@@ -132,6 +132,23 @@ Follow the instructions for your Linux distribution:
   ```
   </Admonition>
 
+  </TabItem>
+  <TabItem label="SUSE Linux Enterprise Server (zypper)">
+
+  Uninstall the Teleport binary using zypper:
+
+  ```code
+  # Change the package name to "teleport" for Teleport Community Edition
+  $ sudo zypper -y remove teleport-ent
+  ```
+
+  Uninstall the Teleport zypper repo:
+
+  ```code
+  $ sudo zypper removerepo teleport
+  ```
+
+  </Admonition>
```

Comment:
```

Error: MDX compilation failed for file "/codebuild/output/src2829177318/src/teleport/docs-website/docs/admin-guides/management/admin/uninstall-teleport.mdx"

154

Cause: Unexpected closing tag `</Admonition>`, expected corresponding closing tag for `<TabItem>` (136:3-136:58)
```

Reply:
Thanks! Fixed in 239150028574c38de7e88265b7e681ccb337aa3b